charts/kusion/README.md

```diff
@@ -22,7 +22,13 @@ Kusion - An Intent-Driven Platform Orchestrator
 
 ## Installing the Chart
 
-> Note: A valid kubeconfig configuration is required for Kusion to function properly. You must either use the installation script, provide your own kubeconfig in values.yaml, or set it through the --set parameter
+> **Important:** Kusion requires a valid kubeconfig configuration to function properly. You can provide it via:
+> - Installation script (recommended)
+> - Custom values.yaml file
+> - Helm --set parameter
+
+> **Note:** Default kubeconfig path is `/var/run/secrets/kubernetes.io/kubeconfigs/`
```

Comment:
The AppVersion in the badge still shows 0.14.0 while the Version shows 0.14.2. These should be consistent to avoid confusion about the actual version being deployed.